Code/Engine/RendererVulkan/Device/Implementation/DeviceVulkan.cpp

```diff
@@ -197,7 +206,10 @@ vk::Result ezGALDeviceVulkan::SelectDeviceExtensions(vk::DeviceCreateInfo& devic
   };
 
   VK_SUCCEED_OR_RETURN_LOG(AddExtIfSupported(VK_KHR_SWAPCHAIN_EXTENSION_NAME, m_extensions.m_bDeviceSwapChain));
-  AddExtIfSupported(VK_EXT_SHADER_VIEWPORT_INDEX_LAYER_EXTENSION_NAME, m_extensions.m_bShaderViewportIndexLayer);
+  if (AddExtIfSupported(VK_EXT_SHADER_VIEWPORT_INDEX_LAYER_EXTENSION_NAME, m_extensions.m_bShaderViewportIndexLayer) != vk::Result::eSuccess)
+  {
+    ezLog::Warning("The optional extension '" VK_EXT_SHADER_VIEWPORT_INDEX_LAYER_EXTENSION_NAME "' is not present");
```

Comment:
I've added a general warning if an extension can't be found in the AddExtIfSupported lambda above so you can remove this block and just call AddExtIfSupported and ignore the return value for optional extensions.